src/screens/DropButton.js

```diff
@@ -42,13 +42,20 @@ const DropButtonPage = () => (
       ]}
       description="A Button that controls a Drop"
       intrinsicElement="button"
-      code={`<DropButton
-  label="Fancy Selector"
-  dropAlign={{ top: 'bottom', right: 'right' }}
-  dropContent={
-    <Box pad="large" background="light-2" />
-  }
-/>`}
+      code={`function Example() {
+  const align = { top: "bottom", left: "right" };
+  const dropAlign = React.useMemo(() => align, [align]);
```

Comment:
I think this will create a new `align` on each render, and this cause `useMemo` to generate a new `dropAlign` on each render. For this example, I think we can just move `align` outside the render function to a constant at the top of the file.

Reply:
yes, when testing i realized this `useMemo` still causes the rerender. i've rewritten it so that it is only called when rendered and tested it on the example from [this isses](https://github.com/grommet/grommet/issues/5928) and it is fixing the rerender.

should we recommend this way or moving the prop outside of the render as a constant? i've updated the stories in storybook in this way: https://github.com/grommet/grommet/pull/6017

Reply:
What about just omitting alignment from the code and relying on the default?